test/parallel_api/algorithm/alg.modifying.operations/remove.pass.cpp

```diff
@@ -105,6 +105,54 @@ struct test_non_const
     }
 };
 
+void test_empty_list_initialization()
```

Comment:
Same basic comment on using `invoke_on_all_policies` here (and throughout the rest of the changes).

Reply:
see above